sync/lib.sh

```diff
@@ -804,20 +804,42 @@ syncLibraryLoaded() {
 
   # reset compatibility variables
   [[ -n "$syncHostServerRoleIndex" ]] && {
-    export SERVERS="${syncHostHostname[$syncHostServerRoleIndex]}"
-    export syncSERVER="${syncHost[$syncHostServerRoleIndex]}"
-    export syncSERVER_IP="${syncHostIP[$syncHostServerRoleIndex]}"
-    export syncSERVERv6="${syncHostIPv6[$syncHostServerRoleIndex]}"
+    SERVERS="${syncHostHostname[${syncHostServerRoleIndex[0]}]}"
+    syncSERVER="${syncHost[${syncHostServerRoleIndex[0]}]}"
+    syncSERVER_IP="${syncHostIP[${syncHostServerRoleIndex[0]}]}"
+    syncSERVERv6="${syncHostIPv6[${syncHostServerRoleIndex[0]}]}"
+    for (( i=1; i<${#syncHostServerRoleIndex[@]}; i++))
+    do
+      SERVERS="$SERVERS ${syncHostHostname[${syncHostServerRoleIndex[$i]}]}"
+      syncSERVER="$syncSERVER ${syncHost[${syncHostServerRoleIndex[$i]}]}"
+      syncSERVER_IP="$syncSERVER_IP ${syncHostIP[${syncHostServerRoleIndex[$i]}]}"
+      syncSERVERv6="$syncSERVERv6 ${syncHostIPv6[${syncHostServerRoleIndex[$i]}]}"
+    done
```

Comment:
```suggestion
    for i in "${syncHostServerRoleIndex[@]}"; do
      SERVERS+=" ${syncHostHostname[$i]}"
      syncSERVER_HOSTNAME+=( "${syncHostHostname[$i]}" )
      syncSERVER+=( "${syncHost[$i]}" )
      syncSERVER_IP+=( "${syncHostIP[$i]}" )
      syncSERVERv6+=( "${syncHostIPv6[$i]}" )
    done
    SERVERS="${SERVERS:1}"
```
The logic here is to be able to use the variable without index the same no matter how many numbers are there.